app/assets/javascripts/discourse/app/templates/composer/dominating-topic.hbs

```diff
@@ -1,12 +1,7 @@
-<a
-  href
-  {{on "click" (fn this.closeMessage this.message)}}
-  class="close"
-  aria-label={{i18n "composer.esc_label"}}
->
-  {{i18n "composer.esc"}}
-  {{d-icon "times"}}
-</a>
+<ComposerTipCloseButton
+  @closeAction={{this.closeMessage}}
+  @message={{this.message}}
```

Comment:
Since that component doesn't do anything (on its own) with the `@message`, I think this would be a cleaner API:

```suggestion
  @action={{fn this.closeMessage this.message}}
```